CONTRIBUTING.rst

```diff
@@ -1,53 +1,47 @@
 Contributing Guidelines
 =======================
 
-To learn more about the purpose of PEPs and how to go about writing a PEP, please
-start reading at PEP 1 (`pep-0001.txt <./pep-0001.txt>`_ in this repo). Note that
-PEP 0, the index PEP, is now automatically generated, and not committed to the repo.
+To learn more about the purpose of PEPs and how to go about writing one, please
+start reading at `PEP 1 <https://www.python.org/dev/peps/pep-0001/>`_.
+Also, make sure to check the `README <./README.rst>`_ for information
+on how to render the PEPs in this repository.
+Thanks again for your contributions, and we look forward to reviewing them!
```

Comment:
"Thanks again for your contributions, and we look forward to reviewing them!" 

Personally I'm not a fan of these statements, as they add little substantive value but add to the text one needs to read. I'd personally not include it, but won't fight the inclusion that strongly!

Reply:
It was there before; I just moved it to the top where it was more appropriate (since it was no longer near the end where it had been) and fixed the phrasing. I can understand a concern with adding too much fluff, but I don't think sparing a single line out of well over 100 to say something welcoming is excessive, and even if not, given it would be removing something that's been in the original for years, I don't believe its worth changing now.